package/network/services/hostapd/files/hostapd.sh

```diff
@@ -1372,14 +1372,12 @@ wpa_supplicant_add_network() {
 
 			key_mgmt="$wpa_key_mgmt"
 
-			if [ ${#key} -eq 64 ]; then
+			if [ "$_w_mode" = "mesh" -o "$auth_type" = "sae" ]; then
```

Comment:
(in case you answer all the question and this became the final code... please use || instead of -o